changelog/fix-4821-link-enabled-capture-later

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add link payment method for capture later option
```

Comment:
Can we tweak the wording? 

As I read, the change is about enabling Link to support authorization/capture scenarios.
And the changelog entry is misleading from this angle.